src/docs/Foundations/Layout/Grid/code.mdx

```diff
@@ -7,15 +7,13 @@ order: 1
 
 ## Basics
 
+<HintItem>Use flexy with a container to create page-level layouts.</HintItem>
 <HintItem>
-  Use flexy with a container to create page-level layouts
+  Use flexy when possible to create flex layouts inside smaller elements or
+  components.
```

Comment:
Can you delete the line break please?

Reply:
@tiloyi The break line is generated by Prettier, i can't have the control :/

Reply:
Oh, okay, I understand
Let's leave it like that, and I'll check on my end to see if there's a way to configure Prettier differently.